src/middlewares/requestValidations.js

```diff
@@ -63,4 +63,28 @@ const validateTrip = (req, res, next) => {
   next();
 };
 
-export default { validateTripRequest, validateTrip };
+/**
+   * @function
+   * @description Validates open trip
+   * @static
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @param {object} next
+   * @returns {object} JSON response
+   */
+const validateOpenTrip = (req, res, next) => {
```

Comment:
The way this is implemented, anyone can edit any request, you need to check if the `userId` of the request is the same as the `id` of the requester
```suggestion
const validateOpenTrip = (req, res, next) => {
const { id } = req.user;
```